target/scripts/postfix-wrapper.sh

```diff
@@ -20,11 +20,11 @@ trap "service postfix stop" SIGINT
 trap "service postfix stop" SIGTERM
 trap "service postfix reload" SIGHUP
 
-service postfix start
-sleep 5
+service postfix start &
+wait
 
 # wait until postfix is dead (triggered by trap)
 while kill -0 "$(< /var/spool/postfix/pid/master.pid)"
 do
-  sleep 5
+  sleep 1
 done
```

Comment:
Inspired by your other changes, I wonder if this one liner would work too:

`wait $(</var/spool/postfix/pid/master.pid)`

Reply:
Unfortunately, `wait` only works on child processes of the current shell:

```
# wait $(</var/spool/postfix/pid/master.pid)
bash: wait: pid 3108 is not a child of this shell 
```

https://stackoverflow.com/a/41613532/568737 suggests `tail` instead (which works):

`tail -f --pid="$(</var/spool/postfix/pid/master.pid)" /dev/null`

We could replace the while loop with that.

Reply:
Sounds good -- including in https://github.com/docker-mailserver/docker-mailserver/pull/2134

Reply:
I am unsure. This could lead to problems on MacOS? I don't have a mac to test, but the stackoverflow posts suggest this only for linux.
On the other hand, we don't support MacOS officially. What do you think @docker-mailserver/maintainers?

Reply:
The code doesn't run on mac, it'll run inside of the linux contianer. Shouldn't be an issue.

Reply:
You are absolutely right.